### PR TITLE
Client helper functions for making calls to the server

### PR DESCRIPTION
**Pull request by brentcolson**

### CODE REVIEW

File: client/services/helper.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+/**
+ *  Access JWT token from user's browser. If user has no token,
+ *  set equal to 'default' which the server will register as invalid
+ *  and redirect to login.
+ */
+var localStorageToken = local.storage.getItem('token') ? local.storage.getItem('token') : 'default';
```
Comment (echo304) [inline client/services/helper.js]: In my opinion, `window.localStorage` would be better choice. [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)

File: client/services/helper.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+/**
+ *  Post helper function for making api calls to server
+ *  @expected arguments - Url, data (i.e. object of user data or product data)
+ *  @return {Object}
+ */
+var postHelper = function(url, data){
+  
+  axios.post(url, {
```
Comment (echo304) [inline client/services/helper.js]: Oops! every helper method don't `return` Promise!

**Comment by echo304** [comment]
Feedback Commented! sry Brent :(

**Comment by brentcolson** [comment]
@echo304 no worries!  is it commented in staging??

**Comment by echo304** [comment]
Commets are on your PR

File: client/services/helper.js (hunk 2 above)
Comment (brentcolson) [inline client/services/helper.js]: Not quite sure what you mean, sorry!  Was it an issue with the comments or with the postHelper? @echo304

File: client/services/helper.js (hunk 1 above)
Comment (brentcolson) [inline client/services/helper.js]: I have updated localStorageToken to use, window.localStorage, should be all set there.

File: client/services/helper.js (hunk 2 above)
Comment (echo304) [inline client/services/helper.js]: `postHelper` and `getHelper` don't `return` `axios.get` and `axios.post`.
It means that `res` returned from axios method will not be passed to its user

**Pull request merged**